src/simplewallet/simplewallet.cpp

```diff
@@ -5011,6 +5021,94 @@ bool simple_wallet::check_spend_proof(const std::vector<std::string> &args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::get_reserve_proof(const std::vector<std::string> &args)
+{
+  if(args.size() != 1 && args.size() != 2) {
```

Comment:
That'd break for: reserve_proof 10 some message

Reply:
I intentionally disallowed the message to include whitespaces (which is also the case in tx/spend proof schemes) because the command argument cannot know the number of consecutive whitespaces if any. I think this is a reasonable restriction.